### PR TITLE
[Improvement-4012][common/remote] Json util code integration,remove the remote module json util

### PR DESCRIPTION
**Pull request by zhuangchong**
#4012

## What is the purpose of the pull request

Json util code integration, remove the remote module json util.

## Brief change log

1.remote model remote JsonSerializer class
2.common model JSONUtils add method

## Verify this pull request

This change added tests and can be verified as follows:
  - *Added JSONUtilsTest to verify the change.*

### CODE REVIEW

**Review by CalvinKirs — CHANGES_REQUESTED** [review]
The pom file of the reomte module still depends on jackson-databind, please delete it?

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java [hunk 1]
```diff
@@ -253,6 +268,24 @@ public static String toJsonString(Object object) {
...
 
+    /**
+     * serialize to json byte
+     *
+     * @param obj object
+     * @param <T> object type
+     * @return byte array
+     */
+    public static <T> byte[] toJsonByteArray(T obj)  {
+        String json = "";
```
Comment (CalvinKirs) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java:279]: Here should be a null operation on obj.

**Comment by zhuangchong** [comment]
> The pom file of the reomte module still depends on jackson-databind, please delete it?

done.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java (hunk 1 above)
Comment (zhuangchong) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java:285]: done.

**Review by CalvinKirs — APPROVED** [review]
+1

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**